src/devices/Pn532/Pn532.cs

```diff
@@ -697,12 +869,8 @@ public byte[] AutoPoll(byte numberPolling, ushort periodMilliSecond, PollingType
         /// <summary>
         /// Set the PN532 as a target, so as a card
         /// </summary>
-        /// <param name="mode"></param>
-        /// <param name="mifare"></param>
-        /// <param name="feliCa"></param>
-        /// <param name="picc"></param>
-        /// <returns></returns>
-        public (TargetModeInitialized modeInialized, byte[] initiator) InitAsTarget(TargetModeInitialization mode, TargetMifareParameters mifare, TargetFeliCaParameters feliCa, TargetPiccParameters picc)
+        public (TargetModeInitialized modeInialized, byte[] initiator) InitAsTarget(TargetModeInitialization mode,
```

Comment:
We probably shouldn't delete the parameter documentation and instead fill it out.

Reply:
I agree, but I have no idea what this does, so I'd rather not write anything than something wrong.